### ISSUE TITLE
Yeelights cannot be found by HUE App/Bridge

### ISSUE THREAD

**Issue by dragonfire5020** [hass-emulated-hue/core#77]
**Describe the bug**

I can't connect my Yeelight Bulbs (1S Color) to the HUE App, they can't be found, even after multiple resets.
The App however found the emulated bridge without problems.

**Steps to Reproduce**

1. Open HUE App
2. Reset Yeelights so that they are flickering
3. Click Search on the App
4. Error: No new devices found
5. Bulbs stop flickering, but aren't connected to the HUE Bridge/App. (Notice that they can be found in the Yeelight App all the time for comparison)

**Expected behavior**

Open the App, search for lights, then add them like in the Yeelight App
I noticed an older issue in which the user had problems with the sound equalizer and he also used Yeelight bulbs so I think they should be compatible

**Logs**

Don't know if that is somehow useful 
```
2021-01-12 19:28:59,685 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/config
2021-01-12 19:29:01,516 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:01,547 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:01,570 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.25', 63625)
2021-01-12 19:29:02,584 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.25', 63625)
2021-01-12 19:29:03,596 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.25', 63625)
2021-01-12 19:29:03,699 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:03,734 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:04,605 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.25', 63625)
2021-01-12 19:29:05,821 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:05,854 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:07,929 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:07,959 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:09,777 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/config
2021-01-12 19:29:10,082 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:10,113 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:12,193 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:12,221 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:14,278 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:14,311 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:16,393 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:16,426 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:18,500 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:18,533 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:19,923 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/config
2021-01-12 19:29:20,745 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:20,777 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:22,841 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:22,870 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:24,945 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:24,976 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:27,054 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:27,081 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:29,140 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:29,174 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:29,982 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/config
2021-01-12 19:29:31,310 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/lights
2021-01-12 19:29:31,342 DEBUG emulated_hue.api -- [192.168.178.21] GET /api/a5808031-8ccf-4cf0-ad5c-9bbf512d9b95/groups
2021-01-12 19:29:33,263 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 45301)
2021-01-12 19:29:33,264 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 45301)
2021-01-12 19:29:35,267 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 47834)
2021-01-12 19:29:35,268 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 47834)
2021-01-12 19:29:37,269 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 36159)
2021-01-12 19:29:37,271 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 36159)
2021-01-12 19:29:52,010 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 52661)
2021-01-12 19:30:04,671 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58969)
2021-01-12 19:30:05,676 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58969)
2021-01-12 19:30:06,481 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:30:06,691 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58969)
2021-01-12 19:30:07,693 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58969)
2021-01-12 19:30:11,481 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:30:16,482 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:30:21,483 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:30:26,485 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:30:33,266 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 59351)
2021-01-12 19:30:33,268 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 59351)
2021-01-12 19:30:35,268 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 40928)
2021-01-12 19:30:35,270 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 40928)
2021-01-12 19:30:37,271 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 47015)
2021-01-12 19:30:37,272 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 47015)
2021-01-12 19:30:52,227 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 59353)
2021-01-12 19:30:53,232 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 59353)
2021-01-12 19:30:54,202 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 50782)
2021-01-12 19:30:54,235 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 59353)
2021-01-12 19:30:55,238 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 59353)
2021-01-12 19:30:59,422 DEBUG emulated_hue.api -- [192.168.178.59] GET /description.xml
2021-01-12 19:31:01,583 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.25', 64202)
2021-01-12 19:31:02,584 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.25', 64202)
2021-01-12 19:31:03,595 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.25', 64202)
2021-01-12 19:31:04,604 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.25', 64202)
2021-01-12 19:31:20,892 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58727)
2021-01-12 19:31:23,905 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58727)
2021-01-12 19:31:26,920 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58727)
2021-01-12 19:31:29,946 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58727)
2021-01-12 19:31:30,816 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 49346)
2021-01-12 19:31:32,950 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58727)
2021-01-12 19:31:33,265 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 45781)
2021-01-12 19:31:33,266 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 45781)
2021-01-12 19:31:35,268 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 56974)
2021-01-12 19:31:35,269 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 56974)
2021-01-12 19:31:35,959 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 58727)
2021-01-12 19:31:36,069 DEBUG emulated_hue.api -- [192.168.178.59] GET /description.xml
2021-01-12 19:31:37,269 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 54978)
2021-01-12 19:31:37,270 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 54978)
2021-01-12 19:31:52,019 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 52661)
2021-01-12 19:32:04,670 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 64243)
2021-01-12 19:32:05,671 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 64243)
2021-01-12 19:32:06,498 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:32:06,680 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 64243)
2021-01-12 19:32:07,683 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.59', 64243)
2021-01-12 19:32:11,498 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:32:16,500 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:32:21,500 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:32:26,501 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.1', 47211)
2021-01-12 19:32:33,267 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 35585)
2021-01-12 19:32:33,268 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 35585)
2021-01-12 19:32:35,269 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 38592)
2021-01-12 19:32:35,270 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 38592)
2021-01-12 19:32:37,270 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 48888)
2021-01-12 19:32:37,271 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.178.33', 48888)
```
.33 = Pi, .59 = Tower, .25 = Laptop

**Hardware Info:**
 - Architecture: ARM
 - Addon version: Emulated Hue (dev)
    - Value of "tag_commit_or_branch" if using dev: master
 - OS: Home Assistant OS
 - Hardware: Raspberry Pi 4 B, 4GB

**Additional context**

Long Story Short: I want to use my Yeelights and my Philipps Ambilight TV together for the Home Entertainment System, that should be possible with this software i guess. I'm happy for any solutions, thank you!
